app/assets/javascripts/maplibre.combinedcontrolgroup.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars */
```

Comment:
not sure if this is THE correct way to handle this.
Eslint does not seem to know about the the (a bit alien to me) `=require` ruby style imports.

There are other occurances of this pattern, but not as many as I would expect, so not sure if this is correct.